addon/templates/components/frost-button.hbs

```diff
@@ -1,3 +1,4 @@
+{{! Template for the frost-button component }}
```

Comment:
It's fine for the generator to include this in the same way that it'll include usage comments (like mirage does).  I just delete them as soon as I move into a file because once you start writing it you don't really need the comments.